src/frontend/fonts/raleway-v12-latin-regular.svg

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Fonts are handled by `file-loader`  (`test: /\.(woff(2)?|ttf|eot|otf)(\?v=[0-9]\.[0-9]\.[0-9])?$/`) SVG files by `url-loader` (`test: /\.(png|jpg|gif|svg)$/i`). The loaders work differently. file-loader will just place the file in the filesystem and makes sure the path is correctly written to the file that is referencing it.
url-loader will check how large the file is and store it in the bundle or in the filesystem. Not sure why the output names are different though. I's assume `url-loader` does not really care about the file name but only looks at the content. So when you load the same file (content) twice it will resolve to the same data that has been inlined before which is probably referenced using the hash (just a guess) 😄 .